plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -90,17 +90,43 @@ public function __invoke( OD_Tag_Visitor_Context $context ): bool {
 		if ( is_null( $element_max_intersection_ratio ) ) {
 			$processor->set_meta_attribute( 'unknown-tag', true ); // Mostly useful for debugging why an IMG isn't optimized.
 		} else {
-			// Otherwise, make sure visible elements omit the loading attribute, and hidden elements include loading=lazy.
+			// TODO: Take into account whether the element has the computed style of visibility:hidden, in such case it should also get fetchpriority=low.
```

Comment:
To be addressed later as part of https://github.com/WordPress/performance/issues/1587